src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.html

```diff
@@ -1 +1,27 @@
-<p>vendor-filter works!</p>
+
+<p style="font-size: 30px;" id="SIc"> Status</p>
```

Comment:
You may want to make a div around each p and check box so that they will move together